spec/support/fixture.rb

```diff
@@ -0,0 +1,32 @@
+begin
+  require 'middleman-core/rack'
+rescue LoadError
+end
+
+
+module Middleman
+  module Fixture
+
+    class << self
+
+      def app &block
```

Comment:
Parenthesis around arguments in function definitions (https://github.com/bbatsov/ruby-style-guide#syntax)